pages/api/auth/[...nextauth].ts

```diff
@@ -107,7 +109,7 @@ export const authOptions = {
 	}
 }
 
-const createUserUpdateObj = (user: User, account: Account | null, db_user?: DbUser) => {
+const createUserUpdateObj = (user: User, account: Account | null, profile: any | null, db_user?: DbUser) => {
```

Comment:
Never use `any` type. You can create your own type, but using `any` defeats the purpose of using TypeScript.